src/references/mobile_device_preview.md

```diff
@@ -0,0 +1,62 @@
+---
+id: mobile_preview
+name: Mobile Device Preview
+title: Mobile Device Preview
+tags:
+  - Reference
+---
+
+# Mobile Device Preview
+
+The **Device Preview** is a feature in the **Core Editor** that simulates how certain aspects of a game will appear on a mobile device. It will give creators an approximate look and feel to simulate the mobile experience for players in landscape orientation.
+
+![!Device Preview](../img/MobileDevicePreview/devicepreview.png){: .center loading="lazy" }
+
+## Device Preview
+
+In the editor toolbar creators will find the **Device Preview Options** button that when clicked on will open up the device preview options.
```

Comment:
Yeah. The option is called that when you hover over it. I agree it doesn't sound right. Am going to drop the "Options" part and hopefully it will be Ok.